apps/foundation-app/package.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Can we add some documentation here about running the foundation-app? I ran into some problems when I ran `ng s --app=foundation-demo` after pulling down for the first time. Since it was missing the foundation sites for the node_modules. Perhaps this is where we use `lerna` to install the child dependencies when we run npm install? Or maybe, we keep the dependencies in the top-level package.json?

Reply:
I added some stuff to lerna.json so it will install all dependencies in core and foundation-app with the `lerna bootstrap` command. I still need to figure out how lerna and publishing to npm works, but I'll do that in a separate branch right after this probably. Added documentation too